Add allowCountries param (#334)
LeleDerGrasshalmi/FortniteEndpointsDocumentation: EpicGamesStore/FreePromos.md

## Query Parameters

`locale`: The requested Language, defaults to english <br/>

> `country`: Show all promos available in the requested Country. Also determines the currency used.
> `allowCountries`: Show all promos available in the requested Countries

Author: ```suggestion
`country`: Determines the price and which currency is shown
`allowCountries`: Show all promos available in the requested Countries
```
